client/src/Poem.jsx

```diff
@@ -10,11 +12,16 @@ PoemLine.propTypes = {
   line: PropTypes.string,
 };
 
+const NBSP = '\u00A0'; // Unicode value for non-breaking space
+
+PoemLine.defaultProps = {
+  line: NBSP,
+};
+
 const Poem = ({ poem }) => {
-  const poemLineNodes = poem.text.split(/\n/).map((line, index) => {
-    const NBSP = '\u00A0'; // Unicode value for non-breaking space
-    return <PoemLine line={(line.trim().length === 0) ? NBSP : line} key={index} />;
-  });
+  const poemLineNodes = poem.text.split(/\n/).map(line =>
+    <PoemLine line={(line.trim().length === 0) ? NBSP : line} key={_.uniqueId()} />,
```

Comment:
Okay, I disabled this rule for this line and added a comment about the intention to have IDs in a database for these lines.